python/tests/utils/utils.py

```diff
@@ -414,3 +415,18 @@ async def delete_acl_username_and_password(client: TGlideClient, username: str):
         return await client.custom_command(
             ["ACL", "DELUSER", username], route=AllNodes()
         )
+
+
+def run_with_timeout(func, timeout, on_timeout=None):
```

Comment:
it should be clear the it runs on a different thread with timeout, add description and optional rename function